client/src/ts/dd_main.ts

```diff
@@ -128,3 +128,15 @@ window.addEventListener('resize', () => {
   speedDial.redraw();
   batteryDial.redraw();
 });
+
+// WebSocket
+const ws = new WebSocket('ws://localhost:8080/ws');
+ws.onmessage = (event) => {
+  const msg = JSON.parse(event.data);
+
+  switch (msg.ID) {
```

Comment:
Do we want to annotate the object we pass around to use lowercase keys when serialized to JSON?